apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -15,6 +15,7 @@
           </v-icon>
         </v-card-text>
       </v-card>
+      <v-checkbox color="primary" label="Viewed" />
```

Comment:
The latest commit to this branch seems like it was committed to the wrong branch. The 'Viewed' checkbox should be in a separate PR.